tests/test-helper.js

```diff
@@ -1,8 +1,14 @@
+import QUnit from 'qunit';
 import Application from 'dummy/app';
 import config from 'dummy/config/environment';
 import { setApplication } from '@ember/test-helpers';
 import { start } from 'ember-qunit';
+import { printResults } from 'ember-a11y-testing/test-support';
 
 setApplication(Application.create(config.APP));
 
+QUnit.done(function () {
+  printResults();
+});
```

Comment:
I'm assuming that the host application would have to do a similar setup to get console output? Is there any benefit to doing it this way as opposed to just getting it for free when using the default reporter? This way we preserve original functionality without introducing extra setup work.

Reply:
@drewlee - Ya, the integration within the `DEFAULT_REPORTER` just above, should ensure that all applications gather the output for the final summary but you are right they won't actually log out (with the current set of changes).

Reply:
Oh, I see what's up. We're now outputting all of the results to the console at the end of the test suite, as opposed to each individual test. I think we'll need to get this documented somewhere as there's likely going to be a lot of people wondering WTF results aren't being output in the console anymore by default.

Reply:
So we removed the old `console.error` calls. They were hard to read, and were interspersed throughout the console output. If you had deprecations, other errors, info, warnings, etc, you'd have a really hard time finding the complete list of violations. This way we avoid that by outputting them all in one `console.group`.

As for not wiring it up automatically, I wanted to avoid tightly coupling our code to `QUnit`, but I can make it work for both `QUnit` and `mocha`, the two currently supported test frameworks. I'd still like to provide a setup function that you explicitly call to get the console output, rather than trying to bind it automatically, since this allows those who don't want the console noise to just not setup the console output. WDYT?

Reply:
Understood from a technical perspective. But from a developer experience perspective, it's a really useful feature that seems strange not to have as a default, especially when it makes debugging the source of failures so much easier. I'm not sure if there would be a valid reason for a consumer not to want it. Personally, I'd prefer to avoid "configuration overhead" for core features (meaning that previously, they just worked out of the box), but ultimately I'll leave that decision up to you. 🙂

Reply:
OK, I added a `setupConsoleLogger` function that can be imported and run in the test-helper.js file, which will bind the output to render on `QUnit.done`. I'm going to omit adding an implementation for mocha, as mocha's root hooks are configured in a different way, and I'd rather leave the functional primitives for app authors to implement on their own.

We'll want to add this direction to the documentation, which I plan on adding at the end of this range of refactorings before we publish.